legal/beitragsordnung.md

```diff
@@ -0,0 +1,20 @@
+# Beitragsordnung
+
+Der Jahresbeitrag wird von den Mitgliedern selbst gewählt. Er beträgt jedoch mindestens
+1. X € für natürliche Personen,
+2. Y € für Institute und sonstige juristische Personen. 
```

Comment:
```suggestion
1. 30 € für natürliche Personen,
2. 500 € für Institute und sonstige juristische Personen. 
```

This is the result of some earlier discussion [https://github.com/oemof/oemof/issues/71#issuecomment-758084221](https://github.com/oemof/oemof/issues/71#issuecomment-758084221)